private/dir.go

```diff
@@ -0,0 +1,55 @@
+package private
```

Comment:
> if language access controls can’t enforce it, we can still warn humans :)

right, I was arguing against Go as well :) but still, either have `pkg/` or move them somewhere else w/o `private` looks better to me!